content/how-to/transaction-process/change-transaction-process-in-template/index.mdx

```diff
@@ -38,55 +38,95 @@ from a background info article about the
 The following guide will help you to customise the process flow in the
 template to match the process in our backend.
 
-## Add the new transaction process configuration
+This guide assumes that you have already followed these instructions to
+add a new process:
+
+- [Create a new transaction process with the Sharetribe CLI](/how-to/transaction-process/create-new-transaction-process-with-cli/).
+
+## Add a listing type that uses the transaction process
 
 The
 [src/config/configListing.js](https://github.com/sharetribe/web-template/blob/main/src/config/configListing.js)
 file lists the listing types actively used by the template, as well as
 the transaction processes related to those types. You need to add a
-configuration for a new transaction process. You can either comment out
-the previous active process definitions (if you only want to use the new
+listing type configuration for a new transaction process into the
+`listingTypes` array to use it in the template.
+
+If you have not added any listing types in code before, the entries in
+the listing type array are all commented out by default. They serve as
+examples of what the listing type configuration looks like.
+
+To add a new listing type, you can either replace or comment out any
+previously active process definitions (if you only want to use the new
 process) or leave them as they are (if you want to allow using multiple
 processes in the same application).
 
-```jsx filename="src/config/configListing.js"
+```js filename="src/config/configListing.js"
+export const listingTypes = [
   {
     listingType: 'nightly-booking',
     label: 'Nightly booking',
     transactionType: {
-      process: 'negotiated-booking',
-      alias: 'negotiated-booking/release-1',
+      process: 'booking-with-negotiation',
+      alias: 'booking-with-negotiation/release-1',
       unitType: 'night',
     },
-  }
```

Comment:
This example would be clearer if listingType and label would match with process name, i.e. listingType: "booking-with-negotiation"